src/libs/PureEpochs.sol

```diff
@@ -27,4 +27,8 @@ library PureEpochs {
     function timeRemainingInCurrentEpoch() internal view returns (uint40) {
         return STARTING_TIMESTAMP + (currentEpoch() * EPOCH_PERIOD) - uint40(block.timestamp);
     }
+
+    function clockMode() internal pure returns (string memory) {
+        return "mode=epoch&epochUnderlyingSource=blockTimestamp&epochStartingTimestamp=1663224162&epochPeriod=1296000&";
```

Comment:
Not sure we need the `&` at the end.
```suggestion
        return "mode=epoch&epochUnderlyingSource=blockTimestamp&epochStartingTimestamp=1663224162&epochPeriod=1296000";
```

Reply:
You're totally right. Should be dropped.